docusaurus/docs/jest/cookbook.md

```diff
@@ -44,13 +44,49 @@ npx knapsack-pro-jest --config=jest.config.e2e.js
 
 ## Generate code coverage reports
 
+Each CI node executes multiple batches of tests in Knapsack Pro [Queue Mode](../overview/index.mdx#queue-mode-dynamic-split), and each batch starts a new Jest process. By default, a coverage report generated by each batch overwrites the previous one, which can lead to incomplete coverage data.
+
+To avoid this, you can configure Knapsack Pro to generate a separate coverage report for each batch by setting the environment variable:
```

Comment:
suggestion

```suggestion
To avoid this, you can configure Knapsack Pro to generate a separate coverage report for each batch by setting the following environment variable:
```